docs/specs/em-annotation-app.json

```diff
@@ -0,0 +1 @@
+{"swagger":"2.0","info":{"description":"Api Documentation","version":"1.0","title":"Api Documentation","termsOfService":"urn:tos","contact":{},"license":{"name":"Apache 2.0","url":"http://www.apache.org/licenses/LICENSE-2.0"}},"host":"localhost:4621","basePath":"/","tags":[{"name":"stored-annotation-set-controller","description":"Stored Annotation Set Controller"},{"name":"annotation-set-search-controller","description":"Annotation Set Search Controller"}],"paths":{"/annotationSets":{"get":{"tags":["stored-annotation-set-controller"],"summary":"helloWorld","operationId":"helloWorldUsingGET","consumes":["application/json"],"produces":["*/*"],"responses":{"200":{"description":"OK","schema":{"type":"object"}}}},"post":{"tags":["stored-annotation-set-controller"],"summary":"Create Annotation Set.","operationId":"createAnnotationSetUsingPOST","consumes":["application/json"],"produces":["*/*"],"responses":{"200":{"description":"Success","schema":{"type":"object"}}}}},"/annotationSets/findAllByDocumentUrl":{"get":{"tags":["annotation-set-search-controller"],"summary":"Retrieve all Annotation Sets associated with a given DM URL","operationId":"findAllAnnotationSetByDocumentUrlUsingGET","consumes":["application/json"],"produces":["*/*"],"responses":{"200":{"description":"Success","schema":{"type":"object"}}}}},"/annotationSets/{id}":{"get":{"tags":["stored-annotation-set-controller"],"summary":"Retrieve Annotation Set instance.","operationId":"retrieveAnnotationSetUsingGET","consumes":["application/json"],"produces":["*/*"],"parameters":[{"name":"id","in":"path","description":"id","required":true,"type":"string"}],"responses":{"200":{"description":"Success","schema":{"type":"object"}}}},"put":{"tags":["stored-annotation-set-controller"],"summary":"Update Annotation Set instance.","operationId":"updateAnnotationSetUsingPUT","consumes":["application/json"],"produces":["*/*"],"parameters":[{"name":"id","in":"path","description":"id","required":true,"type":"string"}],"responses":{"200":{"description":"Success","schema":{"type":"object"}}}},"delete":{"tags":["stored-annotation-set-controller"],"summary":"Delete Annotation Set instance.","operationId":"deleteAnnotationSetUsingDELETE","consumes":["application/json"],"produces":["*/*"],"parameters":[{"name":"id","in":"path","description":"id","required":true,"type":"string"}],"responses":{"200":{"description":"Success","schema":{"type":"object"}}}}}}}
```

Comment:
https://hmcts.github.io/restful-api-standards/#api-naming not for this PR but it's where I got visibility of it.